### DISCUSSION TITLE
is there a way to refresh the dataset in the Windows app?

### DISCUSSION THREAD

**Discussion by pookguy88** [chidiwilliams/buzz#671] [Q&A]
is there an easy way to refresh the Whisper dataset in the Windows app?

**Comment by raivisdejus** [comment]
@pookguy88 If you want to use some newer model, you can use Hugging face whisper type and add some updated model from the huggingface repo. 

Some custom models have been listed here https://github.com/chidiwilliams/buzz/discussions/866
